{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/data/loader.py

```diff
@@ -97,6 +97,29 @@ def load_metadata(key: str, location: str):
     return entity_metadata
 
 
+def load_paf(key: str, location: str) -> pd.DataFrame:
+    try:
+        risk = {
+            # todo add keys as needed
+            data_keys.KEYGROUP.PAF: data_keys.KEYGROUP,
+        }[key]
+    except KeyError:
+        raise ValueError(f'Unrecognized key {key}')
+
```

Comment:
I think you should probably also check that the distribution is dichotomous or polytomous and fail if not, since this won't work for continuous risk factors.

Reply:
raise NotImplementedError